### PR TITLE
33338: Remove usage of laminas\mvc from Setup Wizard page

### PR DESCRIPTION
**Pull request by karyna-t**
Eliminate usage of laminas/mvc from Setup Wizard static page.

Detected more dependencies in other places except Wizard page. Further investigation required to complete the elimination of laminas/mvc as specified in https://github.com/magento/magento2/issues/33338 

### Manual testing scenarios (*)
Setup Wizard page looks the same as before introduced changes.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by andrewbess** [comment]
Hello @karyna-tsymbal-atwix 
Thank you for your PR.
It looks well to me.
I will continue with the tests and will add them ASAP.
After that, I will approve this PR.

**Review by andrewbess — CHANGES_REQUESTED** [review]
It requires to add tests

File: setup/view/magento/setup/index.phtml [hunk 1]
```diff
@@ -6,11 +6,47 @@
 
 // phpcs:disable Magento2.Templates.ThisInTemplate
```
Comment (ihor-sviziev) [inline setup/view/magento/setup/index.phtml]: Looks like this could be removed

**Comment by sivaschenko** [comment]
@karyna-tsymbal-atwix please pay attention that the Platform Health project target branch is `platform-health`

**Comment by andrewbess** [comment]
@magento give me test instance

**Comment by andrewbess** [comment]
@magento run all tests

**Review by andrewbess — APPROVED** [review]
Hello @karyna-tsymbal-atwix 
Your changes are approved.
The MFTF tests will be added separately because it requires additional efforts.
Thank you for your job.

**Review by Den4ik — APPROVED** [review]

**Comment by Den4ik** [comment]
@magento Functional Tests EE, Functional Tests B2B, Functional Tests CE

**Review by eduard13 — APPROVED** [review]

**Comment by andrewbess** [comment]
The external currency service still unavailable.
503-in-console
api-server-status
So, functional tests cannot be passed successfully.

This PR will be merged a little bit later

cc @Den4ik

**Pull request merged**